Core/Authoring/PackageBuilder.cs

```diff
@@ -257,11 +258,11 @@ internal static void ValidateReferenceAssemblies(IEnumerable<IPackageFile> files
 
         private static bool IsPrereleaseDependency(PackageDependency dependency)
         {
-            var versionSpec = dependency.VersionSpec;
+            var versionSpec = dependency.VersionRange;
```

Comment:
guess we should deduplicate this `IsPrereleaseDependency` with `PrereleasePackageDependencyRule.IsPrereleaseDependency` (or in the future), do you agree?